### PR TITLE
sshd service disable ignition

### PR DESCRIPTION
**Pull request by pekramp**
New OCP COREOS ignition script to disable sshd

### CODE REVIEW

**Comment by shawndwells** [comment]
@openscap-ci add to whitelist

**Comment by shawndwells** [comment]
@openscap-ci retest this please

**Comment by shawndwells** [comment]
(using this as a training PR -do not merge)

**Comment by shawndwells** [comment]
Double checked the ``data`` elements by URL decoding. Adding to moderate profile and submitting to retest.

**Comment by shawndwells** [comment]
@pekramp to allow me to cleanup a few things, could you enable maintainers to push to this pull request?

image

**Comment by pekramp** [comment]
> @pekramp to allow me to cleanup a few things, could you enable maintainers to push to this pull request?
> 
> image

it was already checked when i logged in to check 
image

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_ra/ignition/shared.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_ra/ignition/shared.yaml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_redirects/ignition/shared.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_redirects/ignition/shared.yaml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/permissions/restrictions/sysctl_net_core_bpf_jit_harden/ignition/shared.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/permissions/restrictions/sysctl_net_core_bpf_jit_harden/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/selinux/selinux_policytype/ignition/shared.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/selinux/selinux_policytype/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/selinux/selinux_state/ignition/shared.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/selinux/selinux_state/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/selinux/selinux_policytype/ignition/shared.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<<<<<<< HEAD
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+=======
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+>>>>>>> a0c50ca6431816076169b10757503d8d4f261bbf
```
Comment (shawndwells) [inline linux_os/guide/system/selinux/selinux_policytype/ignition/shared.yml:5]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/selinux/selinux_state/ignition/shared.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<<<<<<< HEAD
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+=======
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+>>>>>>> a0c50ca6431816076169b10757503d8d4f261bbf
```
Comment (shawndwells) [inline linux_os/guide/system/selinux/selinux_state/ignition/shared.yml:5]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

**Comment by pekramp** [comment]
> @pekramp: PR needs rebase.

@shawndwells i believe this is related to the changes you were doing to make the build server work as I never touched this file.

**Pull request closed without merging**